src/stylesheets/_variables.scss

```diff
@@ -1,13 +1,28 @@
-$color-bg-primary: #36628B;
-$color-bg-secondary: #72BFCF;
+$color-bg-primary: #36628b;
+$color-bg-secondary: #72bfcf;
 $color-text-primary: #fff;
-$color-text-secondary: #D3E2EB;
+$color-text-secondary: #d3e2eb;
 
+$color-navy-blue: #385f9a;
+$color-gray: rgba(0, 0, 0, 0.25);
+$color-teal: rgba(70, 104, 135, 0.49);
+$color-light-pink: #de98df;
+$color-lavender: #b0b6f7;
+$color-sky-blue: #3792c6;
+$color-light-navy-blue: rgba(67, 113, 183, 0.67);
 
 :root {
-    --color-bg-primary: #{$color-bg-primary};
-    --color-bg-secondary: #{$color-bg-secondary};
+  --color-bg-primary: #{$color-bg-primary};
+  --color-bg-secondary: #{$color-bg-secondary};
 
-    --color-text-primary: #{$color-text-primary};
-    --color-text-secondary: #{$color-text-secondary};
+  --color-text-primary: #{$color-text-primary};
+  --color-text-secondary: #{$color-text-secondary};
+
+  --color-navy-blue: #{$color-navy-blue};
+  --color-gray: #{$color-gray};
+  --color-teal: #{$color-teal};
+  --color-light-pink: #{$color-light-pink};
+  --color-lavender: #{$color-lavender};
+  --color-sky-blue: #{$color-sky-blue};
+  --color-light-navy-blue: #{$color-light-navy-blue};
```

Comment:
Just a note - in the case where we're asked to do a dark version of the website, we might need to consider changing these to more generic names. This is perfectly fine though, and I think they are more descriptive.

Reply:
Oh we aren't doing light/dark mode anymore (most likely) but we that's definitely good to note and if we can definitely update these.